docs/docs/generating-nlu-data.mdx

```diff
@@ -52,6 +52,50 @@ Your assistant will always make mistakes initially, but
 the process of training & evaluating on user data will set your model up to generalize
 much more effectively in real-world scenarios.
 
+### Data Augmentation
+
+When you are creating a new intent in your assistant, you may initially face the
+problem where you do not have enough training examples from your users to create a
+reliable model. In this situation you can use data augmentation to supplement the
+initial training dataset with synthetic examples.
+
+Data augmentation is different from generating data from a script because data
+augmenation is able to create the variability necessary to help a model generalise,
+which a static script cannot do. Nonetheless, messages from real users are *always*
+preferable to synthetic data of any kind. Therefore, you should always remove the
+synthetic data once you have collected a sufficient amount of real user data to train
+an effective model.
+
+ The process for adding augmented data to your model is:
+
+
+#### Recommended Use
+
+1. While reviewing conversations from your users in
+[Rasa X](https://rasa.com/docs/rasa-x/), you identify the need for a new intent.
+2. You want to add this intent to your bot, but you don't have enough real user
+messages yet.
+3. You generate a small amount of paraphrases from the existing real user messages
+with the paraphraser from [https://github.com/RasaHQ/paraphraser](https://github.com/RasaHQ/paraphraser).
```

Comment:
The link is giving me an empty page, I think there's a problem in the repo with `master` vs. `main`.

Reply:
I've ran into a small problem when trying to run it: https://github.com/RasaHQ/paraphraser/issues/4

Reply:
@dakshvar22 can I mark this as resolved?

Reply:
Yes 👍

Reply:
I'll re-test this today, thanks 👍🏻